app/workers/web_push_notification_worker.rb

```diff
@@ -9,7 +9,7 @@ def perform(session_activation_id, notification_id)
     session_activation = SessionActivation.find(session_activation_id)
     notification = Notification.find(notification_id)
 
-    return if session_activation.nil? || notification.nil?
+    return if session_activation.nil? || session_activation.web_push_subscription.nil? || notification.nil?
```

Comment:
hey the "find" above will throw an exception so these guards are unnecessary. the only necessary guard is whether web_push_subscription is nil